app/.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,44 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containers/tree/v0.191.1/containers/docker-existing-docker-compose
+// If you want to run as a non-root user in the container, see .devcontainer/docker-compose.yml.
+{
+	"name": "Existing Docker Compose (Extend)",
```

Comment:
Probably call it "Couchers"

Reply:
Yeah I can do a bit of cleanup with the config files I added. My understanding of how being able to access the container the backend runs in is helpful is that it allows you to set breakpoints and run app.py when you want to debug something.

Reply:
As for how to use this

1. Open the`/app` directory in vscode.
2. Make sure you have both the [`Docker`](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-docker) and [`Remote - Containers`](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-containers) extensions installed.
3. Open the Command Palate (CTRL+SHIFT+P)
4. Search for "Reopen in Container"
5. Navigate to and hit ENTER on the `Remote-Containers: Reopen in Container` option. This will reopen the current directory open in VSCode into a container.